### PR TITLE
Add token list option for autocomplete attribute on inputs

### PR DESCRIPTION
**Pull request by aronerben**
Implements a simplified version with enough leniency to represent the spec:
https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete

https://github.com/ocsigen/tyxml/issues/302

### CODE REVIEW

**Comment by aronerben** [comment]
The tests are still missing and I added a question as a comment

File: lib/html_types.mli [hunk 1]
```diff
@@ -2002,6 +2002,7 @@ type form_attrib =
     | `Method
     | `Name
     | `Target
+    (* TODO [aerben] forms only allow `On and `Off, how to deal with that? *)
```
Comment (Drup) [inline lib/html_types.mli:2005]: To answer your question: What's the behavior of web browser if you use the token list in a form ? Given what I know of html rendering, it's probably accepted silently, so we might as well allow it.

Comment (aronerben) [reply lib/html_types.mli:2005]: I check in one of my own forms in Chromium (Version 102.0.5005.115) and if it's a token list it seems to default to `"on"`. Omitting the attribute also defaults to `"on"` in my browser (so I'm assuming it is, as you said, accepted silently).

**Comment by aronerben** [comment]
> Thanks for the PR! I agree this change is for the best, so I would be inclined to merge with a few more test (and some changelog!) It's technically a breaking change, but it's a rather benign one. :)

Thanks for the review! Does the changelog suffice?

I force-pushed the branch to fix my authorship (I had forgot to set my username for this project). I added handling the case of `` `Tokens [] ``, which should default to `"on"` in my opinion.

**Pull request merged**